files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,537 @@ class AaaCfg(object):
+                    appdb_entry['ipv6_dhcp_client'] = cfgdb_entry['ipv6_dhcp_client']
+                elif (('ipv6_dhcp_client' in cfgdb_entry) and (cfgdb_entry['ipv6_dhcp_client'] != appdb_entry['ipv6_dhcp_client'])):
+                    self.mgmt_intf_dhclient_set(intf_keys[0], "ipv6_dhcp_client", cfgdb_entry['ipv6_dhcp_client'] == "true")
+                    appdb_entry['ipv6_dhcp_client'] = cfgdb_entry['ipv6_dhcp_client']
+
+                if (('ipv4_default_gwaddr' in cfgdb_entry) and ('ipv4_default_gwaddr' not in appdb_entry)):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], cfgdb_entry['ipv4_default_gwaddr'], "ADD")
+                    appdb_entry['ipv4_default_gwaddr'] = cfgdb_entry['ipv4_default_gwaddr']
+                elif (('ipv4_default_gwaddr' in cfgdb_entry) and (cfgdb_entry['ipv4_default_gwaddr'] != appdb_entry['ipv4_default_gwaddr'])):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], appdb_entry['ipv4_default_gwaddr'], "DEL")
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], cfgdb_entry['ipv4_default_gwaddr'], "ADD")
+                    appdb_entry['ipv4_default_gwaddr'] = cfgdb_entry['ipv4_default_gwaddr']
+                elif (('ipv4_default_gwaddr' not in cfgdb_entry) and ('ipv4_default_gwaddr' in appdb_entry)):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], appdb_entry['ipv4_default_gwaddr'], "DEL")
+                    del appdb_entry['ipv4_default_gwaddr']
+
+                if (('ipv6_default_gwaddr' in cfgdb_entry) and ('ipv6_default_gwaddr' not in appdb_entry)):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], cfgdb_entry['ipv6_default_gwaddr'], "ADD")
+                    appdb_entry['ipv6_default_gwaddr'] = cfgdb_entry['ipv6_default_gwaddr']
+                elif (('ipv6_default_gwaddr' in cfgdb_entry) and (cfgdb_entry['ipv6_default_gwaddr'] != appdb_entry['ipv6_default_gwaddr'])):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], appdb_entry['ipv6_default_gwaddr'], "DEL")
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], cfgdb_entry['ipv6_default_gwaddr'], "ADD")
+                    appdb_entry['ipv6_default_gwaddr'] = cfgdb_entry['ipv6_default_gwaddr']
+                elif (('ipv6_default_gwaddr' not in cfgdb_entry) and ('ipv6_default_gwaddr' in appdb_entry)):
+                    self.mgmt_intf_default_gwaddr_set(intf_keys[0], appdb_entry['ipv6_default_gwaddr'], "DEL")
+                    del appdb_entry['ipv6_default_gwaddr']
+
+                self.app_db.set_entry(APP_MGMT_INTF_TABLE, cur_appdb_key, appdb_entry)
+        else:
+            if op == "CREATE" or op == "UPDATE" :
+                self.mgmt_intf_ip_prefix_set(intf_keys[0], intf_keys[1], "ADD")
+                if (('gwaddr' in data) and ((appdb_entry == {}) or ('gwaddr' not in appdb_entry))):
```

Comment:
Why repeat `'gwaddr' in data`, May be we can have a parent if case and put the rest within that

Reply:
Done